packages/wonder-blocks-dropdown/components/multi-select.test.js

```diff
@@ -56,10 +56,10 @@ describe("MultiSelect", () => {
         opener.simulate("keyup", {keyCode: keyCodes.space});
         expect(select.state("open")).toEqual(false);
 
-        // Shouldn't open with enter
+        // Should open with enter
```

Comment:
Interesting that this test case has flipped. So it was by design that enter didn't work before?

Reply:
We want the opener to behave like a button and it wasn't because we were following the listbox behaviour.